### ISSUE TITLE
Indiana Jones and the Emperor's Tomb (Steam version): cinematics not scaled

### ISSUE THREAD

**Issue by legluondunet** [elishacloud/dxwrapper#276]
Hello Elisha,
when you use dxwrapper with "Indiana Jones and the Emperor's Tomb" (Steam version), the cinematics are not scaled.
I enabled D3D8to9 but videos are still to small and not screen centered.
A screen capture of how intro videos played:
![image](https://github.com/elishacloud/dxwrapper/assets/325405/e92d0393-219d-4b56-a192-d3da5ba966d7)

Do you know if it is fixable with dxwrapper?
Thank you for your help.

**Comment by legluondunet** [comment]
A screen capture of how intro videos played:
![image](https://github.com/elishacloud/dxwrapper/assets/325405/e92d0393-219d-4b56-a192-d3da5ba966d7)

Another more important issue: Alt Tab makes this game crash...

**Comment by elishacloud** [comment]
The videos in this game are done with GDI not with Direct3D.  GDI is not easy to scale.

**Comment by elishacloud** [comment]
> Another more important issue: Alt Tab could crash the game...

Are you sure this is related to d3d8to9?  It seems like you are using some tool to scale the game. Maybe remove that and just use d3d8to9.  It is probably whatever tool your using to scale the game causing this crash.

**Comment by legluondunet** [comment]
> Are you sure this is related to d3d8to9?
I didn't say that, I just wanted to share my experience with this game and dxwrapper on my computer. 
The main object here is playing cinematics scaled and centered in fullscreen.

**Comment by elishacloud** [comment]
What dxwrapper settings are you using when you see the Alt+Tab crash?  Are you using other wrappers or plug-ins besides dxwrapper?

**Comment by legluondunet** [comment]
> What dxwrapper settings are you using

I tested again with game + dxwrapper fresh install , I only activated D3D8to9 option.
Now the cinematics played centered:
![image](https://github.com/elishacloud/dxwrapper/assets/325405/9e97b993-6c99-4bf3-a345-367d86ef10af)
On the previous screen capture, it was not centered, I used 
[d3d9]
EnableWindowMode           = 1
FullscreenWindowMode       = 1

>Are you using other wrappers or plug-ins besides dxwrapper?

I'm using Wine, but no other wrappers pr plugins.

new: alt-tab crash happened only if you alt+tab during cinematics, not during playing.
